### PR TITLE
Fix e2e tests for portal

### PR DESCRIPTION
**Pull request by jhellar**
## Motivation
https://issues.jboss.org/browse/RAINCATCH-1253

## Verification
- have mongo and redis servers running
- have raincatcher core and angularjs running
- have selenium server running
  - `cd tests/e2e-ui-tests`
  - `npm install`
  - `npm run wdm:update`
  - `npm run wdm:start`
- start the tests
  - `cd tests/e2e-ui-tests`
  - `npm run test-headless:ui -- --suite=portal`

@twnolan ping

### CODE REVIEW

File: tests/e2e-ui-tests/page-objects/mobile/workflow/main.po.js [hunk 1]
```diff
@@ -46,21 +46,21 @@ var WorkflowPage = function() {
...
-          break;
-        case percentage < 63:
-          value = 50;
-          break;
-        case percentage < 88:
-          value = 75;
-          break;
-        default:
-          value = 100;
+      switch (percentage) {
```
Comment (wtrocki) [inline tests/e2e-ui-tests/page-objects/mobile/workflow/main.po.js:49]: This is example of the code that we should not test. 
That's example step and we can continue that without putting any values there - they assume to be zero. 
We can even improve example step to make some values preselected. 

We should move away with complexity of the steps to really test workflow code.

File: tests/e2e-ui-tests/utils/constants.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  files: {
+    URL: '/files',
+    URL_DETAIL: '/files/detail/'
+  },
+  dateFormat: {
+    BIG_ENDIAN: "big-endian", // date format - 2017/01/21
+    LITTLE_ENDIAN: "little-endian", // date format - 01/21/2017
+    MIDDLE_ENDIAN: "middle-endian" // date format - 21/01/2017
+  },
+  days: [
```
Comment (wtrocki) [inline tests/e2e-ui-tests/utils/constants.js:93]: That's no longer needed. There is no date in the RainCatcher anymore.

**Review by wtrocki — APPROVED** [review]

File: tests/e2e-ui-tests/page-objects/mobile/workflow/main.po.js (hunk 1 above)
Comment (jhellar) [reply tests/e2e-ui-tests/page-objects/mobile/workflow/main.po.js:49]: There is some legacy code, which is not needed, but in tests it is not used (i.e. example steps are not being tested)

File: tests/e2e-ui-tests/utils/constants.js (hunk 2 above)
Comment (jhellar) [reply tests/e2e-ui-tests/utils/constants.js:93]: Same as above - legacy code.

**Comment by jhellar** [comment]
@wtrocki I've created another PR with fixes for mobile part as well https://github.com/feedhenry-raincatcher/raincatcher-angularjs/pull/89. Can we close this PR and possibly merge only the one with all fixes?

**Comment by wtrocki** [comment]
@jhellar I'm open for the best flow that will work for you! Closing. 

PS: you can push commits to PR's - there is no need to open another ones etc. as it will mean more work on your side.

**Pull request closed without merging**